test/integration/MetricsMongoRepoSpec.scala

```diff
@@ -19,8 +19,9 @@ package integration
 import org.mockito.ArgumentMatchers._
 import org.mockito.Mockito
 import org.mockito.Mockito._
-import org.scalatest.mockito.MockitoSugar
+import org.scalatestplus.mockito.MockitoSugar
```

Comment:
why are there multiple import changes like this?

Reply:
MockitoSugar is now deprecated, the recommendation is to use the scalatestplus one.